sonar-apex-plugin/pom.xml

```diff
@@ -37,6 +37,26 @@
             <artifactId>apex-check</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.2</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.easytesting</groupId>
```

Comment:
ApexRulesDefinition line: 76  to "adapt" in this case to recover information from Json files